src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,30 @@
+import { useContext } from 'react';
+import { PeopleFilters } from '../../components/PeopleFilters';
+import { PeopleTable } from '../../components/PeopleTable';
+import { TableContext } from '../../store/TableContextProvider';
+
+export const PeoplePage = () => {
+  const { isLoading, isEmptyMessage } = useContext(TableContext);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="columns is-desktop is-flex-direction-row-reverse">
+          {!isLoading && <PeopleFilters />}
+          <div className="column">
+            <div className="box table-container">
+              <div style={{ display: isEmptyMessage ? 'none' : 'block' }}>
+                <PeopleTable />
+              </div>
```

Comment:
- Avoid inline styles
- Just don't render `PeopleTable` at all when you don't have any

Reply:
i need to render PeopleTable in any case, because it has logic inside that change state isEmptyMessage.  This is a bit of a strange logic, but I had to complete it at the very end. Because I missed this condition during development

Reply:
> i need to render PeopleTable in any case, because it has logic inside that change state isEmptyMessage. This is a bit of a strange logic, but I had to complete it at the very end. Because I missed this condition during development

As I mentioned previously you don't need `isEmptyMessage` at all, so get rid of it, and as I mentioned avoid inline styles